core/src/main/scala/org/apache/spark/scheduler/TaskScheduler.scala

```diff
@@ -101,7 +101,8 @@ private[spark] trait TaskScheduler {
   /**
    * Process a decommissioning executor.
    */
-  def executorDecommission(executorId: String): Unit
+  def executorDecommission(
```

Comment:
nit: don't leave an empty implementation here.

Reply:
yea we should make sure it's implemented

Reply:
Okay. Was just trying to avoid changing a bunch of mock/test implementations of this interface. Will revert back to having this be purely abstract.